src/pages/task/taskTabs/testsTable/LogsColumn.tsx

```diff
@@ -66,10 +84,27 @@ export const LogsColumn: React.FC<Props> = ({ testResult, taskAnalytics }) => {
           Raw
         </Button>
       )}
-    </>
+      {isBeta() && filters && (
+        <Button
+          size="xsmall"
+          as={Link}
+          data-cy="task-history"
+          key="task-history"
+          onClick={() => {
+            taskAnalytics.sendEvent({ name: "Click See History Button" });
+          }}
+          to={getTaskHistoryRoute(projectIdentifier, name, filters)}
+        >
+          See history
```

Comment:
Have we considered labelling this button just "History"? I feel like in general with buttons/links on a page, "See" is implied, so it would be nice to save the space.

Reply:
We have not considered it but lets try it out :)